next/components/forms/widget-components/SelectField/Dropdown.tsx

```diff
@@ -2,6 +2,7 @@ import { EnumOptionsType } from '@rjsf/utils'
 import cx from 'classnames'
 import { FC } from 'react'
 
+import { useClickOutsideHandler } from '../../../utils/ClickOutsideHandler/useClickOutsideHandler'
```

Comment:
why not the one from ts-hooks like here ? https://github.com/bratislava/bratislava.sk/pull/712/files

actually, would be nice to get rid of this custom one in the face of library one existing ,feel free to do it as part of this PR

Reply:
thanks for info, I thought that we use this in project. i will use one from ts-hooks